### PR TITLE
Adding team member

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Adding a new team member

Preview :
https://yashkumarverma.github.io/fossasia.org/team

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
@mariobehling @isuruAb

File: team.html [hunk 1]
```diff
@@ -1418,6 +1418,21 @@ <h1>Developers</h1>
 							</div>
                         <span class="speaker-name">Nicco Kunzmann</span>
                         <span>JavaScript/Python Developer</span>
+                    </div>
+				</div>
+				<div class="col-md-3 col-sm-6 speaker-column">
+                    <div class="speaker">
+                        <div class="image-holder">
```
Comment (akshatnitd) [inline team.html]: Fix indentation

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Remove Conflicts !

**Review by isuruAb — COMMENTED** [review]
@YashKumarVerma  please resolve conflicts

**Comment by YashKumarVerma** [comment]
Whops. Fixing in a few hours

**Comment by YashKumarVerma** [comment]
@isuruAb @0x48piraj @akshatnitd 
Fixed, Resolved, Squashed. 😄

**Review by abishekvashok — APPROVED** [review]

**Review by rupav — APPROVED** [review]

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@YashKumarVerma  why you are adding team member in between .please add in last of team list.

**Comment by YashKumarVerma** [comment]
@ParthS007 I don't know why but adding it to last always gave me merge conflicts.

**Comment by YashKumarVerma** [comment]
@ParthS007 and perhaps this is the only thing i can do 😖

**Comment by rupav** [comment]
@YashKumarVerma check my comments on your task instance. Thanks.

**Comment by YashKumarVerma** [comment]
@rupav Sorry but i have abondoned the task because this PR would take time to merge. :/
I couldn't see your message. 😢

**Comment by YashKumarVerma** [comment]
@rupav got the message from mail. Thanks !

**Comment by YashKumarVerma** [comment]
@akshatnitd @rupav @ParthS007 @abishekvashok @isuruAb 

Closing this. The merge conflicts came on and on. So started everything from scratch:
# Moved to #274

**Pull request closed without merging**